promtimer/cbstats.py

```diff
@@ -448,6 +450,88 @@ def get_user_log(cluster, user, password):
             logging.error('error: can\'t access cluster: {}'.format(err))
             return []
 
+class BackupStatsFiles(Source):
+    """
+    Represents a source of stats data that is a Prometheus instance running against
+    the stats data from cbbackupmgr runs.
+    """
+    def __init__(self, prometheus_tsdb_path, short_name, prometheus_port):
+        super().__init__(prometheus_port)
+        self._short_name = short_name
+        self._prometheus_tsdb_path = prometheus_tsdb_path
+        self._config = None
+
+    def short_name(self):
+        return self._short_name
+
+    def host(self):
+        return '127.0.0.1'
```

Comment:
Could use `@property` here to avoid having to use `bs.short_name()` couldn't we?

Reply:
I'll keep this in mind for the future, but I think with how little I call the getter, it's not going to make much of a difference 😛